docs/cookbook.md

```diff
@@ -229,3 +229,28 @@ The bug here, is that inspect does not truly inspect Postgres and always returns
 the same result. `ldap2pg` will always execute the revoke query, thinking
 `mypriv` is granted to `public`, whatever the actual state of the cluster. It's
 up to you to dig in `pg_catalog.pg_database.datacl` to find SQL GRANT.
+
+
+# ldap2pg as Docker container
+
+Already familiar with Docker and willing to save the setup time you're at the right place.
+
+To run the container simply use the command:
+``` console
+$ docker run dalibo/ldap2pg --help
```

Comment:
I suggest `docker run --rm` for such example. It avoid polluting docker ps -a with one shot container.

Reply:
I didn't know the option, that's nice. Will be added.